src/components/addProductsForm/AddProducts.jsx

```diff
@@ -1,5 +1,181 @@
-import React from "react";
+import React, { useState } from "react";
+import { Form, Button, Row, Col, Container } from "react-bootstrap";
+import "./style.scss";
+import db from "../config/firebaseConfig";
 
 export default function AddProducts() {
-  return <div>add Products</div>;
+  const [hasSizes, setHasSizes] = useState(false);
+  const [getProductBrand, setGetProductBrand] = useState("");
+  const [getProductTitle, setGetProductTitle] = useState("");
+  const [getProductDescription, setGetProductDescription] = useState("");
+  const [getProductCategory, setGetProductCategory] = useState("");
+  const [getProductPrice, setGetProductPrice] = useState("");
+  const [getProductSizes, setGetProductSizes] = useState([]);
+
+  // Add products to the database
+  const addNewProductToDatabase = async (e) => {
+    e.preventDefault();
+    // Getting all images and push them into an array
+    const allImages = [];
+    const firstImage = e.target["firstImage"].value;
+    allImages.push(firstImage);
+    const secondImage = e.target["secondImage"].value;
+    allImages.push(secondImage);
+    const thridImage = e.target["thirdImage"].value;
+    allImages.push(thridImage);
+    const fourthImage = e.target["fourthImage"].value;
+    allImages.push(fourthImage);
+    const allSizes = [];
+    // Getting all sizes and push them into an array
+
+    if (hasSizes === true) {
+      const firstSize = e.target["firstSize"].value;
```

Comment:
so in general, i would say that if you ever have more than two variables you're calling firstX and secondX, it needs to be turned into a loop. two is probably my upper limit.

ask yourself this question: what if you need 5 sizes tomorrow?

revamp this code so that anywhere it is like so, all you would have to do is change a single number to handle any random N case